packages/api/src/router/review.ts

```diff
@@ -95,14 +122,17 @@ export const reviewRouter = createTRPCRouter({
   create: protectedProcedure
     .input(CreateReviewSchemaExternal)
     .mutation(async ({ ctx, input }) => {
-      await ctx.db.insert(schema.review).values(
+      // create the review
+      const reviewTable = await ctx.db.insert(schema.review).values(
```

Comment:
```suggestion
      const review = await ctx.db.insert(schema.review).values(
```
the result isn't the whole table